### PR TITLE
Development sasha -cleaned get started page

### PR DESCRIPTION
**Pull request by sasha-astiadi**
update manual:

cleaned get started page
improved weblet_caprover
and sidebar improval

### CODE REVIEW

File: wiki/sidebar.md [hunk 1]
```diff
@@ -21,12 +21,12 @@
 - [Weblets](weblets_home)
   - [Profile Manager](weblets_profile_manager)
   - [CapRover](weblets_caprover)
-  - [Virtual Machine](weblets_vm)
+  - [Virtual Machines](weblets_vm)
```
Comment (xmonader) [inline wiki/sidebar.md]: Move back to singular

File: wiki/getstarted/getstarted_toc.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+
+- [Install Caprover](tfchain_portal_caprover)
```
Comment (xmonader) [inline wiki/getstarted/getstarted_toc.md]: Caprover has nothing to do with tfchain portal, its qlso documented already in weblets directory

File: wiki/manual3_home_new.md [hunk 3]
```diff
@@ -2,20 +2,40 @@
...
+- [**Get Started**](tfgrid3_getstarted)
 
-## Background Info
+## Help Improve This Manual
 
-- [What's New](grid3_new)
-- [Proof of Utilization](proof_of_utilization_manual)
-- [Definitions](grid3_definitions)
\ No newline at end of file
+**ThreeFold Grid 3.0 Manual** is is a open-sourcedocumentation project currently maintained by ThreeFold Foundation's internal team, and constantly undergoes evolving documentation phases. Help us improve this manual. Documentation grants are available for external documentary contributors. Contact **info@threefold.io** for more information on how to become a contributor.
```
Comment (xmonader) [inline wiki/manual3_home_new.md]: Typo: open Source documentation

File: wiki/tfchain_portal/tfchain_portal_caprover.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+## Your First Tutorial, lets install caprover
```
Comment (xmonader) [inline wiki/tfchain_portal/tfchain_portal_caprover.md]: Remove this file from portal directory directory, check in weblets directory content.

**Review by xmonader — CHANGES_REQUESTED** [review]
Caprover doesn't belong to tfchain portal. its doc exists here https://github.com/threefoldfoundation/info_manual3/blob/development/wiki/weblets/weblets_caprover.md if you want to improve it.

**Comment by sasha-astiadi** [comment]
@xmonader conversation resolved, changes made, please review again

**Pull request closed without merging**